src/openrct2/peep/Staff.cpp

```diff
@@ -187,22 +190,21 @@ bool Staff::IsLocationInPatrol(const CoordsXY& loc) const
     return IsPatrolAreaSet(loc);
 }
 
-bool staff_is_location_on_patrol_edge(Peep* mechanic, const CoordsXY& loc)
+// Check whether the location x,y is inside and on the edge of the
+// patrol zone for mechanic.
+bool staff_is_location_on_patrol_edge(const Peep* mechanic, const CoordsXY& loc)
```

Comment:
Since you are refactoring this make it a member function of the `Staff`. Then there is no need to use the `AsStaff`

Reply:
Also change the function to TitleCase